python/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+import os
+
+setup(
+    name='kfserving',
+    version='0.1.0',
+    author_email='ellisbigelow@google.com',
+    license='../../LICENSE.txt',
+    url='https://github.com/kubeflow/kfserving/model-servers/kfserver',
+    description='Model Server for arbitrary python ML frameworks.',
+    long_description=open('README.md').read(),
+    python_requires='>3.4',
+    packages=find_packages(),
+    install_requires=[
+        "tornado >= 1.4.1",
+        "xgboost == 0.82",
```

Comment:
base package should only include tornado since scikit images may not need xgboost (and possibly vice versa)?

Reply:
I probably need to make separate setup.py's for different frameworks. Alternatively we bundle them all together, but it may introduce unnecessary constraints. I'm not really worried about image size.

Reply:
Check the latest :)